### DISCUSSION TITLE
Bug with the Taxonomy filter

### DISCUSSION THREAD

**Discussion by s4sage** [elementor/elementor#27811] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Bug with the Taxonomy filter: When choosing Tags (or custom taxonomy) in the Taxonomy Filter, it fail to work with "Loop Grid > Query > Include By: Term"

### Describe the solution you'd like.

Fix Bug

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by toineenzo** [comment]
You submitted a feature request, not a bug request
